analyze_test.go

```diff
@@ -435,10 +435,10 @@ func TestAnalyze_zone_sync(t *testing.T) {
 			blockCtx{"stream"},
 			true,
 		},
-		"zone_sync invalid args": {
+		"zone_sync invalid with 2 args": {
 			&Directive{
 				Directive: "zone_sync",
-				Args:      []string{"invalid"},
+				Args:      []string{"invalid", "arg"},
```

Comment:
[Shouldn't this be invalid with any args](https://nginx.org/en/docs/stream/ngx_stream_zone_sync_module.html#zone_sync)?

Reply:
Confirmed with the SE team that the new zone_sync arg is a temporary workaround for a specific customer and should stay undocumented. They'll remove it as soon as the underlying problem is properly solved.
 Restore this unit test.